lakefile.lean

```diff
@@ -2,13 +2,26 @@ import Lake
 
 open Lake DSL
 
+/-- Turn a config option into a Boolean -/
+def enabled (val : Option String) (default : Bool := false) : Bool :=
+  match val with
+  | .some "off" => false
+  | .some "false" => false
+  | .some "on" => true
+  | .some "true" => true
+  | .some _ => not default -- This allows -K=opt to flip the default.
+  | _ => default
+
+-- Selectively choose an option if feature is enabled.
+macro "optArg?" x:ident ws d:ident v:term : term =>
+  `(if enabled (get_config? $x) $d then $v else default)
```

Comment:
This is too much code to go in the lakefile. This should be moved into lake if required.

Reply:
For boolean options I suggest just normalizing everything to `get_config? foo |>.isSome`

Reply:
I'd prefer a simple fix for now rather than more work.  I stripped it down though.